### PR TITLE
Fix navbar at the top of the page

### PR DESCRIPTION
**Pull request by fscholdei**
## Proposed changes

Fix navbar at the top of the page

### Your checklist for this pull request

**Contributor**:
- [x] PR is well described and the description can be used as a commit message on squash
- [x] Related issues linked to PR if existing and labels set
- [ ] New code is covered with unit tests
- [ ] New/updated ui components are tested inside the storybook (module ui-components only) 
- [x] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog`
- [x] Feature has been tested with different permissions
- [ ] Documentation updated (only necessary for new features or changed behaviour)

**Reviewer**:
- [ ] The clean code principles are respected ([CleanCode](https://clean-code-developer.com/virtues/))
- [ ] All new code/logic is implemented on the right spot / "Should this be done here?"
- [ ] UI changes fits into the layout
- [ ] The UI views / components are responsive (mobile views)
- [ ] Correct translations are available

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: scm-ui/ui-components/src/navigation/SecondaryNavigation.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ type CollapsedProps = {
 const SectionContainer = styled.aside`
   position: sticky;
   position: -webkit-sticky; /* Safari */
-  top: 2rem;
+  top: 5rem;
```
Comment (Pilopa) [inline scm-ui/ui-components/src/navigation/SecondaryNavigation.tsx:43]: Isn't the Bulma variable currently set to 3.25rem, not 3rem ?

File: scm-ui/ui-webapp/src/containers/NavigationBar.tsx [hunk 2]
```diff
@@ -42,62 +42,6 @@ const LogoItem = styled.a`
   cursor: default !important;
 `;
 
-const StyledNavBar = styled.nav`
```
Comment (Pilopa) [inline scm-ui/ui-webapp/src/containers/NavigationBar.tsx:45]: Why was this moved ?

Comment (fscholdei) [reply scm-ui/ui-webapp/src/containers/NavigationBar.tsx:45]: Very good hint, thank you!
